napari/utils/_tests/test_theme.py

```diff
@@ -98,6 +101,11 @@ def test_rebuild_theme_settings():
     settings.appearance.theme = "another-theme"
 
 
+@pytest.mark.skip(
+    os.getenv('CI', '0') != '0' and sys.version_info < (3, 9),
+    reason="Testing theme on CI is extremely slow ~ 15s per test."
+    "Skip for now until we find the reason",
+)
 @pytest.mark.parametrize(
```

Comment:
@Carreau Perhaps you could also move the parameterized `color` values inside the function in a `for` loop? That should speed things up too.

Reply:
I prefer to keep it outside, if there is a failure in one of the param then we know which one is failing.